cabal.project

```diff
@@ -77,3 +69,16 @@ allow-newer: servant-multipart-api:servant
 
 -- ghc 9
 allow-newer: tdigest:base
+
+-- ghc 9.2
+allow-newer: basement:base
+allow-newer: memory:base
+allow-newer: openssl-streams:bytestring
+allow-newer: servant-js:base
+allow-newer: servant-js:base-compat
+allow-newer: servant-js:lens
```

Comment:
These lines can be removed since [servant-js was revised](https://hackage.haskell.org/package/servant-js-0.9.4.2/revisions/).